packages/documentation/src/pages/platform/front-end-standards/documented-decisions/teamsite-signin.mdx

```diff
@@ -0,0 +1,7 @@
+
+| | | | |
+| --- | --- | --- | --- |
+|**Last Decision Made:**|Remove signin from Teamsite header| **Decision Date:** | 10/2018 |
+|**Revisit Decision:**| No | | |
```

Comment:
most of the documented decisions have a revisit date but I'm not sure we actually follow that.